native/setup-bootstrap/keiko-setup-bootstrap.windows.test.c

```
@@ -264,6 +264,210 @@ static void test_extended_path(void) {
+static void test_remove_tree_unlinks_junctions_without_following(void) {
+  wchar_t staging[MAX_PATH];
+  wchar_t outside[MAX_PATH];
+  make_temp_dir(staging, MAX_PATH, L"keiko-test-staging");
+  make_temp_dir(outside, MAX_PATH, L"keiko-test-outside");
+
+  wchar_t treasure[MAX_PATH];
+  assert(_snwprintf_s(treasure, MAX_PATH, _TRUNCATE, L"%ls\\treasure.txt", outside) > 0);
+  write_file(treasure, "must survive");
+
+  // A real file inside staging, so the walk has ordinary work to do as well.
+  wchar_t inner[MAX_PATH];
```

Clang (14.0.6)

[warning] variable 'inner' is not initialized

(cppcoreguidelines-init-variables)